docs/examples/daterange/errorMessaging.js

```diff
@@ -0,0 +1,63 @@
+// @flow strict
+import { type Node, useState } from 'react';
+import { DateRange } from 'gestalt-datepicker';
+
+export default function Example(): Node {
+  const [startDate, setStartDate] = useState<Date | null>(null);
+  const [endDate, setEndDate] = useState<Date | null>(null);
+  const [currentEndErrorMessage, setCurrentEndErrorMessage] = useState<
+    [string | null, Date | null] | null,
+  >(null);
+  const [currentStartErrorMessage, setCurrentStartErrorMessage] = useState<
+    [string | null, Date | null] | null,
+  >(null);
+  const [endErrorMessage, setEndErrorMessage] = useState<string | null>(null);
+  const [startErrorMessage, setStartErrorMessage] = useState<string | null>(null);
+
+  const date = new Date();
+  const year = date.getFullYear();
+
+  const minDate = new Date(year, 6, 1);
+  const maxDate = new Date(year, 6, 31);
+
+  return (
+    <DateRange
+      endDateValue={endDate}
+      endDateErrorMessage={endErrorMessage}
+      minDate={minDate}
+      maxDate={maxDate}
+      onStartDateBlur={() => {
```

Comment:
Just wondering, any reason we aren't handling error message internally? I'm not sure I follow what the blur is doing? Seems like a lot of handlers have to be defined externally just to get it to show

Reply:
DONE